src/app/people/page.tsx

```diff
@@ -1,5 +1,226 @@
+  font-size: 1.5rem;
+  color: ${Color.gray900};
+`
+
+const SpecialThanksText = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const SpecialThanksSubtitle = styled.h3`
+  ${FontVariant.body_lg}
+  color: ${Color.gray900};
+  margin: 0;
+`
+
+const SpecialThanksDescription = styled.p`
+  ${FontVariant.body_md}
+  color: ${Color.gray700};
+  gap: 4px;
+`
 
 export default function Page() {
-  return <main></main>
+  const [sidebarList, setSidebarList] = useState<string[]>([])
+  const [activeSection, setActiveSection] = useState<string | null>(null)
+  const ignoreObserver = useRef(false)
+  const sectionRefs = useRef<Record<string, HTMLElement | null>>({})
+
+  useEffect(() => {
+    const observer = new IntersectionObserver(
+      entries => {
+        if (ignoreObserver.current) return
+
+        entries.forEach(entry => entry.isIntersecting && setActiveSection(entry.target.id))
+      },
+      {
+        // when a section passes the area that is 10% from the top and 80% from the bottom of the viewport, it will be considered intersecting
+        rootMargin: '-20% 0px -80% 0px',
+      }
+    )
+
+    const sections = Object.values(sectionRefs.current)
+    sections.forEach(section => {
+      section && observer.observe(section)
+    })
+
+    return () => {
+      sections.forEach(section => {
+        if (section) {
+          observer.unobserve(section)
+        }
+      })
+    }
+  }, [])
+
+  useEffect(() => {
+    handleSidebarList(MEMBERS_KEY)
+  }, [])
+
+  const handleSidebarList = (list: string[]) => {
+    const sections = CareerTypes.filter(career => career !== 'Alumni').map(career =>
+      career.replace(/\s+/g, '').replace('.', '')
+    )
+    sections.push('alumni')
+    setSidebarList(sections)
+  }
+
+  const handleLinkClick = (sectionId: string) => {
+    setActiveSection(sectionId)
+    ignoreObserver.current = true
+
+    setTimeout(() => {
+      ignoreObserver.current = false
+    }, 1000)
+  }
+
+  return (
+    <Container>
+      <main style={{ padding: '0px', margin: '0px' }}>
+        <h1>People</h1>
+        <Sections>
+          {CareerTypes.filter(career => career !== 'Alumni').map(
+            (career, i) =>
+              MEMBERS_KEY.filter(key => MEMBERS[key].career === career && career !== 'Alumni').length >= 1 && (
+                <React.Fragment key={career}>
+                  <Section
+                    key={career}
+                    id={career.replace(/\s+/g, '').replace('.', '')}
+                    ref={el => {
+                      sectionRefs.current[career] = el
+                    }}
+                  >
+                    <SectionTitle id={career.replace(/\s+/g, '').replace('.', '')}>{career}</SectionTitle>
+                    <SectionContent>
+                      {MEMBERS_KEY.filter(key => MEMBERS[key].career == career && career != 'Alumni').map(key => (
+                        <MemberCard key={key} mem={MEMBERS[key]} />
+                      ))}
```

Comment:
Related to my comment I left last time, we need to add a sorting algorithm that we have in the prev version, not just get the order from the MEMBERS variable, as it will be changed as people are added

Reply:
[2ae2871](https://github.com/kixlab/website/pull/40/commits/2ae28718c6c5186ec6e5be196ffe2576ed8fa3f4)
Had to change the sorting algorithm slightly for alumni, since not all of them have an end date. The updated algorithm sorts them by their start date.